README.md

```diff
@@ -52,12 +52,12 @@
 # OKD clusters in the container with libvirt
 
 * `okd-base` contains all needed packages to provision and run OKD cluster on top of the libvirt provider
-* `okd-4.1.0-rc.0` okd-4.1.0-rc.0 cluster provisioned with OpenShift installer on top of the libvirt provider
+* `okd-4.1` okd-4.1 cluster provisioned with OpenShift installer on top of the libvirt provider
 
 ## Versions to use
 
-* `kubevirtci/okd-base`: `sha256:918d3c7f7c5ec94057715897f589c11b38e74c80927ee5af857e24817baeebaf`
-* `kubevirtci/okd-4.1.0-rc.0`: `sha256:71afc34d9299aa11313b43a584e7e9d7e2f962279453b53d853a9d3bcb8b3255`
+* `kubevirtci/okd-base`: `sha256:90b0522eed6dc2593300b33b05977d3a2d30581e58f05943658791c87d2bae89`
+* `kubevirtci/okd-4.1.0`: `sha256:e90cb3690753bc053f85537e531f6242e5d31f5504f17a619d8dc4b72e3ba622`
```

Comment:
This image only worked after i modified it to be kubevirtci/okd-4.1:sha256.... instead of kubevirtci/okd-4.1.0:sha256....

i think the image got uploaded wiht the old okd-4.1 prefix